RELEASE-NOTES.txt

```diff
@@ -1,6 +1,6 @@
 20.9
 -----
-
+* [*] Login Flow: Provide ability for user to cancel login flow from Me screen [#19349]
```

Comment:
I think we could add more details here, this will be edited for clarity if needed before being included in the final release notes. 
I was thinking about "Provide ability for user to cancel login WP.com flow when already logged in to a self-hosted site", wdyt?

Reply:
Sure - I've updated release notes with your suggestion